src/main/java/teammates/storage/api/CommentsDb.java

```diff
@@ -388,9 +388,9 @@ public void deleteCommentsByInstructorEmail(String courseId, String email) {
 
         List<Comment> giverComments = this.getCommentEntitiesForGiver(courseId, email);
         // for now, this list is empty
-//        List<Comment> recipientComments = this.getCommentEntitiesForRecipients(courseId,
-//                CommentRecipientType.INSTRUCTOR, email);
-//        getPM().deletePersistentAll(recipientComments);
+        // List<Comment> recipientComments = this.getCommentEntitiesForRecipients(courseId,
```

Comment:
not sure whether this is a good practice?  Would you please cite a source for this. As in I understand comments should align with the code, but here is commented out code instead of java comments and changing the indentation feels as if the commented out code is actually explanatory